roles/tas_single_node/tasks/podman/rekor.yml

```diff
@@ -25,6 +25,7 @@
       systemd_file: redis
       network: "{{ tas_single_node_podman_network }}"
       kube_file_content: "{{ lookup('ansible.builtin.template', 'manifests/rekor/redis-server.yaml') | from_yaml }}"
+  when: tas_single_node_rekor_redis.database_deploy
```

Comment:
We should also use this for conditionally (not) pulling the image in `tasks/podman.yml`. (If I'm looking right, we should also improve the condition for prepulling the trillian_db image with `tas_single_node_trillian.database_deploy` - could you please address that as well?)
Thanks!